tools/gnur-runtime/src/main/java/org/renjin/gnur/api/Rdynload.java

```diff
@@ -62,12 +66,14 @@ public static boolean R_forceSymbols(DllInfo info, boolean value) {
 //   DL_FUNC R_FindSymbol (char const *, char const *, R_RegisteredNativeSymbol *symbol)
 //
 
-  public static void R_RegisterCCallable (BytePtr packageName, BytePtr name, Object method) {
-    // This is intended to allow other packages to call this method. 
-    // We'll ignore this for the moment.
+  public static void R_RegisterCCallable (BytePtr packageName, BytePtr name, MethodHandle method) {
+    String key = packageName.toString(packageName.getArray().length-1) + "." + name.toString(name.getArray().length-1);
```

Comment:
This won't necessarily always work. C strings are null-terminated: it can be that a much longer memory chunk is allocated but only the first bytes used, for example, and terminated with a null, in which case subsequent calls will fail. Use `BytePtr.nullTerminatedString()` to convert to a `java.lang.String` instance.